tiledb/bioimg/converters/base.py

```diff
@@ -203,12 +210,24 @@ def from_tiledb(
 
             with slide, writer:
                 writer.write_group_metadata(slide.properties)
-                for level in slide.levels:
-                    if level < level_min:
+                group_metadata = jsonpickle.loads(
+                    slide.properties.get("metadata", "{}")
+                )
+                for idx, _ in enumerate(slide.levels):
+                    if idx < level_min:
                         continue
-                    level_image = slide.read_level(level, to_original_axes=True)
-                    level_metadata = slide.level_properties(level)
-                    writer.write_level_image(level, level_image, level_metadata)
+                    level_image = slide.read_level(idx, to_original_axes=True)
+                    level_metadata = {
+                        "group_metadata": group_metadata,
+                        "writer_metadata": slide.level_properties(idx),
+                    }
+                    writer.write_level_image(
+                        idx,
+                        idx == level_min,
```

Comment:
Why do you need an extra argument in the function `baseline` to perform this condition check when both parts of the condition are being passed into the function?